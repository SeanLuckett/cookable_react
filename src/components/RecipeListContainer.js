import React, { Component } from 'react';

class RecipeListContainer extends Component {
  render() {
    return(<RecipeList/>);
  }
}

export default RecipeListContainer;