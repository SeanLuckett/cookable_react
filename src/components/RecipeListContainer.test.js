import React from 'react';
import { shallow } from 'enzyme';
import RecipeListContainer from './RecipeListContainer';

describe('RecipeListContainer', () => {
  const container = shallow(<RecipeListContainer/>);

  it('renders', () => {
    expect(container).toMatchSnapshot();
  });

  it('contains a RecipeList component', () => {
    expect(container.find('RecipeList').exists()).toBe(true);
  });
});