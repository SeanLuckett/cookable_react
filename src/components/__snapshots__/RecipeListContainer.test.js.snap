// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeListContainer renders 1`] = `<RecipeListContainer />`;
